backend/src/main/java/com/happy/friendogly/club/service/ClubCommandService.java

```diff
@@ -120,4 +121,22 @@ public UpdateClubResponse update(Long clubId, Long memberId, UpdateClubRequest r
         }
         throw new FriendoglyException("수정 권한이 없습니다.", HttpStatus.FORBIDDEN);
     }
+
+    public void kickMember(Long clubId, Long memberId, DeleteKickedMemberRequest request) {
+        Club club = clubRepository.getById(clubId);
+        Member member = memberRepository.getById(memberId);
+        Member kickedMember = memberRepository.getById(request.kickedMemberId());
+        validateKick(club, member, kickedMember);
+        club.removeClubMember(kickedMember);
+        club.removeChatRoomMember(member);
+    }
+
+    private void validateKick(Club club, Member owner, Member kickedMember) {
```

Comment:
```suggestion

    private void validateKick(Club club, Member owner, Member memberToKick) {
```

이름 제안해봅니다. 과거형 쓰면 메서드 호출 시점에 이미 kick 되었다는 느낌이 들 것 같아요.

Reply:
과거형이라기보다 수동태 생각하고 쓴 느낌인데 좀 이상한가요?

Reply:
수동태라고 해도 member to be kicked가 되어야 할 것 같네요. 아직 강퇴당한 상태가 아니니까요!